tracer/test/test-applications/integrations/Samples.OracleMDA/Samples.OracleMDA.csproj

```diff
@@ -11,16 +11,22 @@
 
   </PropertyGroup>
 
-  <ItemGroup>
-    <PackageReference Include="Oracle.ManagedDataAccess" Version="$(ApiVersion)" />
+  <ItemGroup Condition=" '$(TargetFrameworkIdentifier)' != '.NETFramework' ">
+    <PackageReference Include="Oracle.ManagedDataAccess.Core" Version="$(ApiVersion)"/>
```

Comment:
I kept this the same as before, but I think this `Version="$(ApiVersion)"` is bullshit, because Oracle versions numbering is not aligned on dotnet versions. But I'm not sure what to put instead, so in doubt I left it like this.